doc/handle_scope.md

```diff
@@ -43,8 +43,8 @@ operator HandleScope::napi_handle_scope
 operator Napi::HandleScope::napi_handle_scope() const
 ```
 
-Returns the N-API napi_handle_scope wrapped by the `Napi::EscapableHandleScope` object.
-This can be used to mix usage of the C N-API and node-addon-api by allowing
+Returns the Node-API napi_handle_scope wrapped by the `Napi::EscapableHandleScope` object.
```

Comment:
Could you use backtick for napi_handle_scope? From napi_handle_scope to `napi_handle_scope`